libs/novui/src/components/button/Button.tsx

```diff
@@ -21,7 +20,7 @@ type IconButtonDefaultElement = 'button';
 
 export type IButtonProps<C extends React.ElementType = IconButtonDefaultElement> = PolymorphicComponentPropWithRef<
```

Comment:
Since this is a type and not an interface, there is no need for the `I` prefix.
```suggestion
export type ButtonProps<C extends React.ElementType = IconButtonDefaultElement> = PolymorphicComponentPropWithRef<
```